elements/pfe-navigation/demo/demo.css

```diff
@@ -0,0 +1,74 @@
+[slot="logo"] > a {
+  color: white;
+  text-decoration: none;
+}
+
+.redhat-logo {
+  max-width: 110px;
+  max-height: 36px;
+}
+
+@media (min-width: 992px) {
+  .redhat-logo {
+    max-width: 135px;
+  }
+}
+
+.site-title {
+  position: relative;
+  line-height: 48px;
+  font-size: .75rem;
+  text-transform: uppercase;
+  letter-spacing: .025em;
+  text-decoration: none;
+}
+
+.site-title:before {
+  position: absolute;
+  left: 20px;
+  top: 6px;
+  content: "";
+  height: 34px;
+  width: 1px;
+  background: rgba(255,255,255,0.3)
+}
+
+.gray {
+  background-color: #ccc;
+  margin-top: 16px;
+  padding: 16px;
+}
+
+.container {
+  grid-template-columns: 1fr;
+  max-width: var(--pfe-navigation--Width);
+  margin: 0 auto;
+  display: grid;
+  text-align: left;
+}
+
+.test-overflow {
+  height: 100vh;
```

Comment:
Is this to show that if theres more content than vertical space for the tray, its still scrollable?

Reply:
Exactly!  :) I wanted to have a good-enough solution for that edge case because we've seen it happen before with navigation